src/android/FileUtils.java

```diff
@@ -614,7 +617,11 @@ private boolean hasReadPermission() {
     }
 
     private boolean hasWritePermission() {
-        return PermissionHelper.hasPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE);
+          if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                return true;
```

Comment:
```suggestion
                // Starting with API 33, requesting WRITE_EXTERNAL_STORAGE is an auto permission rejection
                // Apps now have read-write access to it's own (created) files, so we can assume `true`, however not all files are readable.
                // See https://github.com/apache/cordova-plugin-file#androids-external-storage-quirks for more details.
                return true;
```

Reply:
Just to explain why we are returning true for future contributors/code maintainers